CHANGELOG/v1.7.3.md

```diff
@@ -0,0 +1,49 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.2
+## :chart_with_upwards_trend: Overview
+- 18 new commits merged
+- 1 feature addition ✨
+- 5 bugs fixed 🐛
+
+## :sparkles: New Features
+- ClusterClass: Support ControlPlane variable overrides (#10701)
+
+## :bug: Bug Fixes
+- ClusterClass: Defaulting webhook should check class is set in ClusterClass-based clusters (#10673)
+- ClusterClass: Fix indexing in ClusterClass webhook MHC validation (#10678)
+- MachineHealthCheck: Skip publishing the `RemediationRestricted` event when there are no unhealthy target (#10616)
+- MachinePool: Check that replicas pointer is not nil in machinepool controller (#10634)
+- KCP/CAPD: KCPTemplate & DockerClusterTemplate webhook: default before immutability check (#10646)
+
+## :seedling: Others
+- clusterctl: Bump cert-manager to v1.15.0 (#10732)
+- Dependency: Bump to Go 1.21.11 (#10740)
+- Devtools: Use setup-envtest from CR main (05/2024) to use envtest binaries from CT releases (#10663)
+- e2e: Kind: bump to v0.23.0 (#10641)
+- e2e: Test: check for metadata.yaml when resolving releases to not try to use unreleased versions (#10645)
+- e2e: Test: improve output for ValidateResourceVersionStable by using BeComparable instead of Equal (#10654)
+- Security: Cleanup owner files (#10679)
+- Testing: Support MachinePools without MachinePoolMachines in clusterctl upgrade test (#10614)
+- Testing: Use kind as a secondary management cluster for clusterctl E2E tests (#10639)
```

Comment:
```suggestion
```
Same comment as on the other PR (https://github.com/kubernetes-sigs/cluster-api/pull/10743#discussion_r1634537788)

/hold

Reply:
/hold cancel
Double checked all other PRs, the other ones are from release-1.6. But to be honest I don't have the time to do this :)

Reply:
Updated the PR